config-wdio/wdio.local.conf.ts

```diff
@@ -118,7 +129,7 @@ export const config: WebdriverIO.Config = {
   // Define all options that are relevant for the WebdriverIO instance here
   //
   // Level of logging verbosity: trace | debug | info | warn | error | silent
-  logLevel: "info",
+  logLevel: "silent",
```

Comment:
Silent log? can we change it back to Info

Reply:
Hi @balualtimetrik  , with info log, each and every background event is shown on the console which is not required. Silent will keep the events minimal which is required for debug hence as informed keeping the same.